packages/react-dom/src/events/plugins/__tests__/ModernSimpleEventPlugin-test.js

```diff
@@ -231,15 +231,14 @@ describe('SimpleEventPlugin', function() {
   describe('interactive events, in concurrent mode', () => {
     beforeEach(() => {
       jest.resetModules();
-
       ReactDOM = require('react-dom');
       Scheduler = require('scheduler');
     });
 
     // @gate experimental
     it('flushes pending interactive work before extracting event handler', () => {
       container = document.createElement('div');
-      const root = ReactDOM.unstable_createRoot(container);
+      const root = ReactDOM.createRoot(container);
```

Comment:
I didn't change this one. Git just thinks I renamed it whereas I actually deleted old test.

Kind of confusing why it uses unprefixed name. I thought that doesn't work anymore.